src/scenes/home/profile/profile.js

```diff
@@ -56,10 +56,12 @@ const Profile = ({ verified }) => (
               Get Verified for Added Benefits
             </span>
             <p>
-              In order to take advantage of resources such as Mentorship and Scholarships, we
-              require that you verify your military affiliation status. We use Id.Me for
-              verification, a highly secure and specialized platform that will protect your
-              information. We will not store or transfer any of your sensitive information.
+              To gain access to <span className={styles.profileContainer__verified__services}>Mentorship</span> and <span className={styles.profileContainer__verified__services}>Scholarships</span>, we
```

Comment:
I think we need to clarify that we are open to spouses too. Something like, "require that you verify your status as a veteran, servicemember, or military spouse."

Reply:
@hollomancer - How about something to the effect of:

```html
<p>Veterans, service members, spouses - sign in below to get verified!</p>
```

Reply:
Include servicemembers as a single word, and we're good!

Reply:
NOOOO don't forget the 'and': `<p>Veterans, service members, and spouses - sign in below to get verified!</p>`

Reply:
I don't feel that 'and' is necessary ... similar to 'Friends, Romans, countrymen, lend me your ears' - the less verbage, the better.

Reply:
@hollomancer @jjhampton @ksmacleod99 - Thoughts on this:

![image](https://user-images.githubusercontent.com/7482329/35783471-f17e7724-09c4-11e8-9860-dccfc5f33d1c.png)

Reply:
I'm good with it.